src/engraving/layout/layoutsystem.cpp

```diff
@@ -1246,6 +1246,7 @@ void LayoutSystem::processLines(System* system, std::vector<Spanner*> lines, boo
     if (segments.size() > 1) {
         //how far vertically an endpoint should adjust to avoid other slur endpoints:
         const qreal slurCollisionVertOffset = 0.65 * system->spatium();
+        const qreal slurCollisionHorizOffset = 0.15 * system->spatium();
```

Comment:
What does this number mean?

Reply:
That's the amount that the slurs move in opposite directions horizontally so as not to be running into each other